deployment/terraform/container_service.tf

```diff
@@ -309,6 +319,19 @@ data "template_file" "app_cc" {
   }
 }
 
+resource "aws_ecs_task_definition" "app_cc" {
```

Comment:
@vladsha-dev it seems GitHub confuses blocks related to app_cc and  app_logstash. Could you please somehow make them in a order, hither clearly shows what exactly was added